3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/outputs/EnrichedEvent.scala

```diff
@@ -133,9 +133,14 @@ class EnrichedEvent {
   @BeanProperty var tr_total: String = _
   @BeanProperty var tr_tax: String = _
   @BeanProperty var tr_shipping: String = _
+  @BeanProperty var tr_currency: String = _
   @BeanProperty var tr_city: String = _
   @BeanProperty var tr_state: String = _
   @BeanProperty var tr_country: String = _
+  @BeanProperty var tr_total_base: String = _
+  @BeanProperty var tr_tax_base: String = _
+  @BeanProperty var tr_shipping_base: String = _
+  @BeanProperty var currency_base: String = _
```

Comment:
Can you add another field please called currency_rate_at: String, populated with EOD_PRIOR in the EnrichmentManager. Thanks!